pkg/types/service_plan.go

```diff
@@ -101,7 +102,9 @@ func (e *ServicePlan) Validate() error {
 			return fmt.Errorf("service plan metadata is invalid JSON")
 		}
 	}
-
+	if time.Duration(e.MaximumPollingDuration)*time.Second > 7*24*time.Hour {
```

Comment:
is this value coming from OSB spec or we chose it ?

Reply:
We choose it

Reply:
does it make sense for it to be configurable? is there a situation in which we want to make it smaller while troubleshooting an issue or as a mitigation of a "buggy" situation? I cant think of an example thats why this is more of a question